src/components/core/Sidenav.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Please check to see how it is implemented, there seems to be a problem.

Reply:
Working on them.